src/robot/reporting/jsbuildingcontext.py

```diff
@@ -20,13 +20,15 @@
 from robot.utils import (attribute_escape, get_link_path, html_escape,
                          html_format, is_string, is_unicode, timestamp_to_secs,
                          unic)
+from .expandkeywordmatcher import ExpandKeywordMatcher
 
 from .stringcache import StringCache
```

Comment:
The added import should be moved one row down.

Reply:
fixed